content/blog/2019/root-transition/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: User guide for extenting Istio root certificate lifetime
+description: The user guide to extend Istio root certificate lifetime.
+publishdate: 2019-06-06
+subtitle:
+attribution: Oliver Liu
+twitter:
+keywords: [security, PKI, certificate, Citadel]
+
+---
+
+## Background
+
+For Istio releases prior to 1.0.8 and prior to 1.1.8, the Istio self-signed certificates have the default lifetime of 1 year.
```

Comment:
You m,ight try `For istio releases 1.0.0 - 1.1.8 inclusive,`

Reply:
This doesn't quite make sense, prior to 1.0.8 and prior to 1.1.8 by definition is just prior to 1.1.8 (that covers 1.0.x)

Reply:
When I originally read this, I thought it meant that 0.8 didn't suffer from this problem.  in responding to your review, I found the error of my thinking.1.0.9 will not have this problem for example.  The wording is still a little clunky and hard to process.  I just woke up so I don't have any brilliance atm :)

Reply:
Sorry for the confusion. Reworded.